src/overlays/actors/ovl_Bg_Ddan_Jd/z_bg_ddan_jd.c

```diff
@@ -41,7 +41,7 @@ typedef enum {
     STATE_GO_BOTTOM,
     STATE_GO_MIDDLE_FROM_BOTTOM,
     STATE_GO_MIDDLE_FROM_TOP,
-    STATE_GO_TOP,
+    STATE_GO_TOP
```

Comment:
The trailing comment is necessary to prevent clang from formatting this enumeration like so: 

`typedef enum { STATE_GO_BOTTOM, STATE_GO_MIDDLE_FROM_BOTTOM, STATE_GO_MIDDLE_FROM_TOP, STATE_GO_TOP } BgDdanJdState;`

Reply:
trailing comma actually induces a warning on IDO
putting offset comments (which we should have anyway) will prevent this formatting